ansible/roles/nginx-proxy/tasks/main.yml

```diff
@@ -51,3 +51,10 @@
       - env={{ env }}
       - suffix!={{ suffix }}
   include_tasks: remove-objects.yml
+
```

Comment:
The config should be created before a nginx is deployed. Please rearrange the order of tasks.